LiskHQ/lisk-hub
# Implement monitor header - Closes #2507

### What issue have I solved?

Closes #2507

### How have I implemented/fixed it?

- [x] Add "Monitor" to top bar
- [x] Update top bar to hide unactive menu item labels
- [x] Create Switcher toolbox component 
- [x] Create Switcher component demo
- [x] Create monitor header
- [x] Add "Monitor" icon assets

Note that I introduced one new dependency with MIT licence: https://github.com/jest-community/jest-watch-toggle-config/blob/master/LICENSE that I added not to waste time waiting for test coverage when writing tests but being able to quickly switch the coverage on.

### How has this been tested?

- Navigate main menu items
- Check Monitor header

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
